utils/utils.go

```diff
@@ -0,0 +1,62 @@
+package utils
```

Comment:
Since these transformations are only used in the parquet package, we should probably move them there and not expose them. Each public function we expose as a maintenance cost which is unnecessary in this case.